### PR TITLE
Middleware edit fixes

### PR DESCRIPTION
**Pull request by ddcesare**
Some fixes aiming to help the usage of the middleware edit view.

### CODE REVIEW

**Comment by ddcesare** [comment]
apitools_middleware_edit

**Comment by ddcesare** [comment]
apitools_middleware_edit2

**Comment by mikz** [comment]
@ddcesare should be fixed in master, please rebase this when the build is green

File: app/views/middlewares/edit.html [hunk 1]
```diff
@@ -55,13 +55,15 @@ <h3 class='middleware-edit-title' ng-show="!editTitle">{{ middleware.name }}</h3
         <div pane-set-height split-panes="vertical" class="last-trace-container">
           <div class='last-trace-request' ng-show="last_trace">
             <div class="last-trace-call-box">
-              <h4 class="last-trace-call">{{ last_trace.req.method }} {{ last_trace.req.uri }}</h4>
+              <h4 class="last-trace-call">{{ last_trace.req.method }}{{ last_trace.req.uri }}</h4>
```
Comment (mikz) [inline app/views/middlewares/edit.html]: why there is no space between method and the uri ?
it looks weird imho

**Comment by mikz** [comment]
@ddcesare the redo should also load the latest request


good job! it looks good

**Pull request merged**